public/javascripts/global.js

```diff
@@ -129,14 +129,15 @@ function fetchQuiz(event) {
     var target = $(event.target);
     if (target.hasClass('quiz-btn')){
       var team = $('.selected-team'),
-          id = team.data('id');
+          id = team.data('team-id'),
```

Comment:
Well, probably better to key off the RB id.. but if we do that, I have to
carry the API id with it everywhere bc I need that to make any updates.

If we switch to a different API, I'll have to tweak a bunch of things
around anyways so I'm not that concerned about this for now.

On Mon, Feb 9, 2015 at 10:41 PM, Blake Parkinson <notifications@github.com>
wrote:

> In public/javascripts/global.js
> <https://github.com/blakeparkinson/sports2/pull/82#discussion_r24391886>:
>
> > @@ -129,14 +129,15 @@ function fetchQuiz(event) {
> >      var target = $(event.target);
> >      if (target.hasClass('quiz-btn')){
> >        var team = $('.selected-team'),
> > -          id = team.data('id');
> > +          id = team.data('team-id'),
>
> I think that makes sense for now. I'm still leery of being tied to the API
> ID. If a record of all of the teams is in mongo (which they should be, no?)
> wouldn't it be better to key off of the RB id, and then fetch an update
> from the API depending on our rules?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/blakeparkinson/sports2/pull/82/files#r24391886>.
>



-- 
*Kelsey Gross*
Edmodo Customer Success
https://www.edmodo.com/kelseygross